my-react-app/src/app/screens/Game/index.js

```diff
@@ -30,17 +28,7 @@ class Game extends Component {
       status = `Next player: ${this.props.xIsNext ? 'X' : 'O'}`;
     }
 
-    return (
-      <div className={styles.game}>
-        <div className={styles.gameBoard}>
-          <Board squares={current.squares} onClick={i => this.props.handleClick(i)} />
-        </div>
-        <div className={styles.gameInfo}>
-          <div>{status}</div>
-          <ol>{moves}</ol>
-        </div>
-      </div>
-    );
+    return <Game onMoves={this.moves} />;
```

Comment:
Does this work? You are passing a `onMoves` prop but you're not using it on the layout.